MAUI/DataGrid/Getting-started.md

```diff
@@ -25,25 +25,44 @@ Before proceeding, ensure the following are in place:
 
 ## Step 1: Create a New MAUI Project
 
-### Visual Studio
+{% tabcontents %}
+{% tabcontent Visual Studio %}
 
 1. Go to **File > New > Project** and choose the **.NET MAUI App** template.
 2. Name the project and choose a location. Then, click **Next**.
 3. Select the .NET framework version. Then, click **Create**.
 
-### Visual Studio Code
+{% endtabcontent %}
+{% tabcontent Visual Studio Code %}
 
 1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and **Enter**.
 2. Choose the **.NET MAUI App** template.
 3. Select the project location. Then, type the project name and **Press Enter**.
```

Comment:
3. Select the project location. Then, type the project name and press **Enter**.